pmm-app/src/pmm-dbaas/components/DBCluster/PSMDB.service.ts

```diff
@@ -78,6 +79,8 @@ const toAPI = (dbCluster: DBCluster) => ({
       compute_resources: {
         cpu_m: dbCluster.cpu * 1000,
         memory_bytes: dbCluster.memory * 10 ** 9,
+        // disk_size units - Gigabytes
+        disk_size: Number(dbCluster.disk),
```

Comment:
Why we need to cast to Number?

Reply:
to consistency with other fields, I'm not sure what type will we expect on a backend

Reply:
But the field is already a number as I see in the interface, so you are casting a number to a number :D

Reply:
Not really, it's a string because technically when we fill this field it gives us a string. other fields such as CPU also strings but we cast it when multiplying

Reply:
Oh okay, you are right, thanks for the explanation!